commands/inspect_builder.go

```diff
@@ -48,60 +49,61 @@ func InspectBuilder(logger *logging.Logger, cfg *config.Config, client PackClien
 	return cmd
 }
 
-func inspectBuilderOutput(logger *logging.Logger, client PackClient, imageName string, local bool) {
+func inspectBuilderOutput(logger logging.LoggerWithWriter, client PackClient, imageName string, local bool) {
+	out := logger.Writer()
 	info, err := client.InspectBuilder(imageName, local)
 	if err != nil {
 		logger.Error(err.Error())
 		return
 	}
 
 	if info == nil {
-		logger.Info("\nNot present")
+		_, _ = fmt.Fprintln(out, "\nNot present")
 		return
 	}
 
 	if info.Description != "" {
-		logger.Info("\nDescription: %s", info.Description)
+		logger.Infof("\nDescription: %s", info.Description)
 	}
 
-	logger.Info("\nStack: %s\n", info.Stack)
+	_, _ = fmt.Fprintf(out, "\nStack: %s\n\n", info.Stack)
```

Comment:
Same question as above about using `Infof` instead. Here and pretty much everywhere there is a `fmt.Fprintln`.